agent/browser/ie/wptbho/wptbho.cc

```diff
@@ -95,6 +97,24 @@ STDMETHODIMP_(void) WptBHO::OnDocumentComplete(IDispatch *pDisp,
 }
 
 /*-----------------------------------------------------------------------------
+Called when navigation is done
+-----------------------------------------------------------------------------*/
+STDMETHODIMP_(void) WptBHO::OnNavigateComplete(IDispatch *pDisp,
+  VARIANT * vUrl) {
+  CString url;
+  if (vUrl)
+    url = *vUrl;
+  AtlTrace(_T("[WptBHO] OnNavigateComplete - %s"), url);
+
+
+  CComPtr<IUnknown> unknown_browser = _web_browser;
+  CComPtr<IUnknown> unknown_frame = pDisp;
+  if (unknown_browser && unknown_frame && unknown_browser == unknown_frame) {
+    _wpt.NavigateComplete2(url);

```

Comment:
Why do we invoke NavigateComplete2 on OnNavigateComplete? Shouldn't it just be NavigateComplete?